micmac_li3ds/import_orimatis.py

```diff
@@ -156,261 +207,159 @@ def get_acquisition_datetime(node):
-
-    @staticmethod
-    def get_flatfield(node):
-        flatfield = node.findtext('auxiliarydata/stereopolis/flatfield_name')
-        return flatfield.strip() if flatfield else None
-
-    @staticmethod
-    def get_date(node):
-        date = xmlutil.child_int(node, 'auxiliarydata/stereopolis/date')
-        Y = 2000 + int(date/10000)
-        m = int(date/100) % 100
-        d = date % 100
-        return datetime.date(Y, m, d).isoformat()
+        xmlutil.child_check(node, 'time_system', 'UTC')
+        return datetime.datetime(Y, m, d, H, M, S, s, pytz.UTC)
 
     @staticmethod
-    def get_project(node):
-        tag = 'auxiliarydata/stereopolis/chantier'
-        return xmlutil.child(node, tag).text.strip()
+    def get_calibration_datetime(root):
+        tag = 'geometry/intrinseque/sensor/calibration_date'
+        date = xmlutil.finddate(root, tag, ['%d-%m-%Y', '%m-%Y'])
+        return date.replace(tzinfo=pytz.UTC) if date else None
 
-    @staticmethod
-    def get_position(node):
-        tag = 'auxiliarydata/stereopolis/position'
-        return xmlutil.child(node, tag).text.strip()
-
-    @staticmethod
-    def get_session(node):
-        return xmlutil.child_int(node, 'auxiliarydata/stereopolis/session')
 
-    @staticmethod
-    def get_section(node):
-        return xmlutil.child_int(node, 'auxiliarydata/stereopolis/section')
-
-    @staticmethod
-    def get_numero(node):
-        return xmlutil.child_int(node, 'auxiliarydata/stereopolis/numero')
-
-    def get_or_create_camera_sensor(self, node):
-        name = self.basename
+class Sensor(Api.Sensor):
+    def __init__(self, sensor, node, metadata):
         pixel_size = None
-        serial = ''
-
         if node.tag == 'sensor':
-            name = xmlutil.child(node, 'name').text.strip()
             pixel_size = xmlutil.child_float(node, 'pixel_size')
-            serial = xmlutil.child(node, 'serial_number').text.strip()
 
         image_size = xmlutil.child_floats(node, 'image_size/[width,height]')
-        specs = {
-            'image_size': image_size,
-            'pixel_size': pixel_size,
-            'flatfield': self.get_flatfield(node),
-        }
-        return self.api.get_or_create_sensor(
-            name=self.sensor_name or name,
-            sensor_type='camera',
-            sensor_id=self.sensor_id,
-            description='imported from "{}"'.format(self.basename),
-            serial=serial,
-            specs=specs,
+        super().__init__(
+            sensor,
+            specifications={
+                'image_size': image_size,
+                'pixel_size': pixel_size,
+                'flatfield': metadata.get('flatfield'),
+            },
         )
 
-    def get_or_create_wo_referential(self, node, sensor):
-
-        node = xmlutil.child(node, 'geometry/extrinseque')
-        systeme = xmlutil.child(node, 'systeme').text.strip()
-        grid_alti = xmlutil.child(node, 'grid_alti').text.strip()
 
+class Referential:
```

Comment:
Please do not use classes as namespaces. I think you don't need a namespace here. You can just add functions to the `import_orimatis` module.